bz2/bz2.php

```diff
@@ -137,7 +138,12 @@ function bzerror($bz): array {}
  * @return string|int The compressed string, or an error number if an error occurred.
  */
 #[Pure]
-function bzcompress(string $data, int $block_size = 4, int $work_factor = 0): string|int {}
+function bzcompress(
+    string $data,
+    #[PhpStormStubsElementAvailable(from: '5.3', to: '7.0')] int $blocksize,
+    #[PhpStormStubsElementAvailable(from: '7.1')] int $block_size = 4,
```

Comment:
$block_size doesn't have a default value till 8.0.
7.4 result:
```
Parameter name: string(6) "source"
Default value unavailable
Parameter name: string(9) "blocksize"
It is optional.
Default value unavailable
Parameter name: string(10) "workfactor"
It is optional.
Default value unavailable
```

vs 8.0

```
Parameter name: string(4) "data"
Default value unavailable
Has paramter type: array(1) {
  [0]=>
  string(6) "string"
}
Parameter name: string(10) "block_size"
It is optional.
Default Value available. It is: int(4)
Has paramter type: array(1) {
  [0]=>
  string(3) "int"
}
Parameter name: string(11) "work_factor"
It is optional.
Default Value available. It is: int(0)
Has paramter type: array(1) {
  [0]=>
  string(3) "int"
}
```

Reply:
Yep, but still, it was optional that's why I had to add a default value to it.